### DISCUSSION TITLE
✅ 🚀 Feature Request: List name of Icon-box in the Navigator

### DISCUSSION THREAD

**Discussion by vehrmann** [elementor/elementor#20768] [Feature Request]
Hi elementor folks,
It would be amazing if you could list the name of icon-boxes in the navigator.
As I have several icon-boxes which differ only slightly but which are used multiple times throughout my page, it would come in handy to see which one is actually used on the current page.
Thanks a lot for considering this feature :)

Best
Vincent

**Comment by nicholaszein** [comment]
Hello @vehrmann!

Thank you for contributing with your idea! 🙏 

### ✅ This feature already exists since the Navigator/Structure Panel was introduced in version 2.2.0!

All you need to do is double-click on the item title and define your own name.

Please refer to the [documentation](https://elementor.com/help/navigator/) for a better understanding of what can be done with the navigator.

### 🔎 In the future, make sure you have searched the [Academy](https://elementor.com/academy), and [Help Center](https://elementor.com/help) before submitting your 🚀 Feature Request to save time!

Kind regards
